<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="6" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-07-11T15:58:31Z" name="Suite" finished-at="2022-07-11T15:58:31Z" duration-ms="66">
    <groups>
      <group name="dev1">
        <method signature="GroupTest.dev2()[pri:1, instance:com.test.tests.GroupTest@1ab06251]" name="dev2" class="com.test.tests.GroupTest"/>
      </group> <!-- dev1 -->
      <group name="dev">
        <method signature="GroupTest.dev1()[pri:0, instance:com.test.tests.GroupTest@1ab06251]" name="dev1" class="com.test.tests.GroupTest"/>
        <method signature="GroupTest.dev2()[pri:1, instance:com.test.tests.GroupTest@1ab06251]" name="dev2" class="com.test.tests.GroupTest"/>
        <method signature="GroupTest.dev3()[pri:2, instance:com.test.tests.GroupTest@1ab06251]" name="dev3" class="com.test.tests.GroupTest"/>
      </group> <!-- dev -->
    </groups>
    <test started-at="2022-07-11T15:58:31Z" name="Test1" finished-at="2022-07-11T15:58:31Z" duration-ms="66">
      <class name="com.test.tests.GroupTest">
        <test-method signature="dev1()[pri:0, instance:com.test.tests.GroupTest@1ab06251]" started-at="2022-07-11T15:58:31Z" name="dev1" finished-at="2022-07-11T15:58:31Z" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dev1 -->
        <test-method signature="dev2()[pri:1, instance:com.test.tests.GroupTest@1ab06251]" started-at="2022-07-11T15:58:31Z" name="dev2" finished-at="2022-07-11T15:58:31Z" duration-ms="4" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.test.tests.GroupTest.dev2(GroupTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dev2 -->
        <test-method signature="dev3()[pri:2, instance:com.test.tests.GroupTest@1ab06251]" started-at="2022-07-11T15:58:31Z" name="dev3" depends-on-groups="dev1" finished-at="2022-07-11T15:58:31Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method GroupTest.dev3()[pri:2, instance:com.test.tests.GroupTest@1ab06251] depends on not successfully finished methods in group "dev1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method GroupTest.dev3()[pri:2, instance:com.test.tests.GroupTest@1ab06251] depends on not successfully finished methods in group "dev1"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dev3 -->
      </class> <!-- com.test.tests.GroupTest -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
